src/main/java/com/letsparty/web/controller/PartyController.java

```diff
@@ -181,34 +194,97 @@ public String modify(@PathVariable int partyNo, @Valid PartyForm partyForm, Bind
 	}
 	
 	@PreAuthorize("isAuthenticated()")
-	@GetMapping("/{partyNo}/post")
+	@GetMapping("/{partyNo}/post-form")
 	public String addPost(@PathVariable int partyNo, @AuthenticationPrincipal LoginUser loginUser, Model model) {
 		UserPartyApplication upa = userPartyApplicationService.findByPartyNoAndUserId(partyNo, loginUser.getId());
 		model.addAttribute("upa", upa);
 		PostForm postForm = new PostForm();
 		model.addAttribute("postForm", postForm);
-		return "/page/party/post";
+		return "/page/party/post-form";
 	}
 	
 	// 게시물 제출
 	@PreAuthorize("isAuthenticated()")
-	@PostMapping("/{partyNo}/post")
+	@PostMapping("/{partyNo}/post-form")
 	public String addPost(@PathVariable int partyNo, @AuthenticationPrincipal LoginUser loginUser, @Valid PostForm postForm,
 			BindingResult error, Model model) {
 		UserPartyApplication upa = userPartyApplicationService.findByPartyNoAndUserId(partyNo, loginUser.getId());
 		model.addAttribute("upa", upa);
 		// TODO 제목, 본문 글자 수 제한하는 유효성 구현하기
 		if (error.hasErrors()) {
-			return "page/party/post";
+			return "page/party/post-form";
 		}
 		partyService.insertPost(postForm, partyNo, loginUser.getId());
 		return "redirect:/party/{partyNo}";
 	}
 	
 	@GetMapping("/{partyNo}")
-	public String home(@PathVariable int partyNo, Model model) {
-		// partyNo를 사용하여 파티 게시물을 조회합니다.
-		return "page/party/home";
+	public String home(@PathVariable int partyNo) {
+		// partyNo를 사용해서 마지막 게시글 번호를 불러온다.
+		int postNo = postService.getLastPostNoByPartyNo(partyNo);
+		return "redirect:/party/{partyNo}/read/" + postNo;
+	}
```

Comment:
```suggestion
	public String home(@PathVariable int partyNo) {
		// partyNo를 사용해서 마지막 게시글 번호를 불러온다.
		int postNo = postService.getLastPostNoByPartyNo(partyNo);
		if (postNo == 0) {
			return "page/party/home";
		}
		return "redirect:/party/{partyNo}/read/" + postNo;
	}
```
글이 없는 경우를 제 의도와는 다른 방법으로 구현하셨네요. 전달이 되지 않았을까봐 제 의도를 전체 코드로 적어둡니다.
제안을 이해했었지만 의도적으로 다른 방법으로 하신 거라면 그대로 진행하셔도 될 것 같아요.